metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java

```diff
@@ -27,31 +29,53 @@
 import com.netflix.metacat.common.server.connectors.exception.PartitionAlreadyExistsException;
 import com.netflix.metacat.common.server.connectors.exception.TableAlreadyExistsException;
 import com.netflix.metacat.common.server.usermetadata.UserMetadataServiceException;
-import com.netflix.servo.monitor.DynamicCounter;
-import com.netflix.servo.monitor.DynamicTimer;
-import com.netflix.servo.monitor.Stopwatch;
-import com.netflix.servo.tag.BasicTagList;
-import com.netflix.servo.tag.TagList;
+import com.netflix.spectator.api.Id;
+import com.netflix.spectator.api.Registry;
+import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
 
+import javax.inject.Singleton;
+import javax.validation.constraints.NotNull;
 import javax.ws.rs.core.Response;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
 
+
 /**
  * Request wrapper.
  */
 @Slf4j
+@Singleton
 public final class RequestWrapper {
-    private RequestWrapper() {
+    private final Registry registry;
+
+    //Metrics
+    private final Id requestCounterId;
+    private final Id requestTimerId;
+    private final Id requestFailureCounterId;
+
+    /**
+     * Wrapper class for processing the request.
+     *
+     * @param registry registry
+     */
+    @Inject
+    public RequestWrapper(@NotNull @NonNull final Registry registry) {
+        this.registry = registry;
+        requestCounterId = registry.createId("dse.metacat.counter.requests");
```

Comment:
I think we should take the chance now to change these metrics names from `dse.*` to just `metacat.*`